projects/fsastorefrontstyles/scss/overrides/main-overrides.scss

```diff
@@ -64,3 +64,14 @@ cx-fs-cms-custom-container {
   padding: 0;
   box-shadow: none;
 }
+
+header .SiteLinks {
+  @include media-breakpoint-down(md) {
+    background-color: inherit;
+    position: absolute;
+    right: 7%;
+  }
+  @include media-breakpoint-down(sm) {
+    right: 13%;
+  }
```

Comment:
Doesn't seem a good approach. Perhaps use `calc` with percentage and pixels combination. Then you wouldn't need another breakpoint...